frontend/src/components/OrderSummary.tsx

```diff
@@ -45,15 +48,20 @@ const OrderSummary = ({ restaurant, cartItems, removeFromCart }: Props) => {
                 className="cursor-pointer"
                 onClick={() => removeFromCart(item._id)}
               />
-              ${item.price * item.quantity}
+              ${((item.price * item.quantity) / 100).toFixed(2)}
             </span>
           </div>
         ))}
 
         <Separator />
         <div className="flex justify-between">
           <span>Delivery</span>
-          <span>${restaurant.deliveryPrice}</span>
+          <span>
+            $
+            {cartItems.length > 0
+              ? (restaurant.deliveryPrice / 100).toFixed(2)
+              : 0}
+          </span>
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/omkardongre/Food-Ordering-App/pull/47#issuecomment-2430019945>#d33549</a><span> - 10/22/2024, 07:15 pm</span>
<br>

<div><small>🟠</small> Medium importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> Consider simplifying the conditional rendering of the delivery price. The current implementation might be unnecessarily complex for this use case.
</div>
<div id="fix">
<b>Fix:</b> Consider using a simpler conditional expression or a helper function to determine the delivery price.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -59,6 +59,3 @@
        <span>
 -            $
 -            {cartItems.length > 0
 -              ? (restaurant.deliveryPrice / 100).toFixed(2)
 -              : 0}
 +            ${cartItems.length > 0 ? (restaurant.deliveryPrice / 100).toFixed(2) : '0.00'}
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged